### PR TITLE
Gradient boosting classifier

### PR DESCRIPTION
**Pull request by bkowshik**
## Objective
- Fixes: https://github.com/mapbox/gabbar/issues/11
- Jupyter notebook: `notebooks/GradientBoostingClassifier.ipynb`

### Features of changeset used
- Features created
- Features modified
- Features deleted
- Area of changeset bbox
- Total user changesets
- Total user features

### Algorithm
- [sklearn.ensemble.GradientBoostingClassifier](http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingClassifier.html)

#### Classification report
```
                    precision    recall  f1-score   support

problematic         0.99      0.64      0.77      1522
not problematic     0.95      1.00      0.97      9956

avg / total         0.95      0.95      0.95     11478
```
Metric from cross-validation: `0.95`

learning-curve

### CODE REVIEW

**Comment by bkowshik** [comment]
Closing in favor of: https://github.com/mapbox/gabbar/pull/30
